src/zkbob/ZkBobPoolUSDCMigrated.sol

```diff
@@ -28,8 +28,7 @@ contract ZkBobPoolUSDCMigrated is ZkBobPool, ZkBobTokenSellerMixin, ZkBobUSDCPer
             _tree_verifier,
             _batch_deposit_verifier,
             _direct_deposit_queue,
-            1, // Make sure that TOKEN_NUMERATOR is set in 1000 in ZkBobPool and ZkBobDirectDepositQueue
-            1_000_000_000
+            1 // Make sure that TOKEN_NUMERATOR is set in 1000 in ZkBobPool and ZkBobDirectDepositQueue
```

Comment:
Looks like there is no difference between `ZkBobPoolUSDC` and `ZkBobPoolUSDCMigrated` now. So, we can leave only `ZkBobPoolUSDC`